sonic_platform_base/sfp_base.py

```diff
@@ -16,6 +16,41 @@ class SfpBase(device_base.DeviceBase):
     # Device type definition. Note, this is a constant.
     DEVICE_TYPE = "sfp"
 
+    # Generic error types definition
+    SFP_STATUS_INITIALIZING = 'Initializing'
+    SFP_STATUS_OK = 'OK'
+    SFP_STATUS_UNPLUGGED = 'Unplugged'
+    SFP_STATUS_DISABLED = 'Disabled'
+    SFP_ERROR_DESCRIPTION_BLOCKING = 'Blocking Error'
+    SFP_ERROR_DESCRIPTION_POWER_BUDGET_EXCEEDED = 'Power budget exceeded'
+    SFP_ERROR_DESCRIPTION_I2C_STUCK = 'Bus stuck (I2C data or clock shorted)'
+    SFP_ERROR_DESCRIPTION_BAD_EEPROM = 'Bad or unsupported eeprom'
+    SFP_ERROR_DESCRIPTION_UNSUPPORTED_CABLE = 'Unsupported cable'
+    SFP_ERROR_DESCRIPTION_HIGH_TEMP = 'High temperature'
+    SFP_ERROR_DESCRIPTION_BAD_CABLE = 'Bad cable (module/cable is shorted)'
+
+    # SFP status
+    SFP_STATUS_BIT_REMOVED = 0
+    SFP_STATUS_BIT_INSERTED = 1
+    # SFP error status
+    SFP_ERROR_BIT_BLOCKING = 2
```

Comment:
Maybe hex is more readable here, or could you please add comment at least?

Reply:
Changed to hex.